src/send/SendEnterAmount.tsx

```diff
@@ -158,6 +159,13 @@ function SendEnterAmount({ route }: Props) {
       // should never happen because button is disabled if send is not possible
       throw new Error('Send is not possible')
     }
+
+    if (origin === SendOrigin.Jumpstart) {
+      // TODO handle send transaction and navigation
+      return
+    }
+
+    const recipient = route.params.recipient
```

Comment:
what's the reason behind moving the `recepeint` declaration here?

Reply:
ah, type guard 😅

Reply:
yes 🙈 typescript doesn't let me destructure this from the route.params when it may not exist